diag_manager/fms_diag_axis_object.F90

```diff
@@ -137,6 +140,8 @@ module fms_diag_axis_object_mod
      TYPE(fmsDiagAttribute_type),allocatable , private :: attributes(:) !< Array to hold user definable attributes
      INTEGER                        , private :: num_attributes  !< Number of defined attibutes
      INTEGER                        , private :: domain_position !< The position in the doman (NORTH, EAST or CENTER)
+     integer                        , private :: structured_ids(2) !< If the axis is in the unstructured grid,
```

Comment:
these are the mappings from unstructured to structured, correct?

Reply:
these are not mappings, they are the axis IDs of the original structured axis